### PR TITLE
Name the ride window-invalidate flag

### PR DESCRIPTION
**Pull request by kevinburke**

### CODE REVIEW

**Comment by IntelOrca** [comment]
Can you also create an enum for this and name all the values.

File: src/windows/ride.c [hunk 1]
```diff
@@ -3360,8 +3360,8 @@ static void window_ride_maintenance_update(rct_window *w)
 	widget_invalidate(w, WIDX_TAB_4);
 
 	ride = GET_RIDE(w->number);
-	if (ride->var_14D & 20) {
-		ride->var_14D &= ~20;
+	if (ride->window_invalidate_flags & 20) {
+		ride->window_invalidate_flags &= ~20;
```
Comment (duncanspumpkin) [inline src/windows/ride.c]: This should be 0x20 not 20.

File: src/windows/ride.c [hunk 2]
```diff
@@ -2950,8 +2950,8 @@ static void window_ride_operating_update(rct_window *w)
 	widget_invalidate(w, WIDX_TAB_3);
 
 	ride = GET_RIDE(w->number);
-	if (ride->var_14D & 10) {
-		ride->var_14D &= ~10;
+	if (ride->window_invalidate_flags & 10) {
+		ride->window_invalidate_flags &= ~10;
```
Comment (duncanspumpkin) [inline src/windows/ride.c]: This should be 0x10 not 10.

**Comment by duncanspumpkin** [comment]
I think for the bottom toolbar I called these flags `DIRTY_FLAG`. Although invalidate flag is probably more descriptive.

**Comment by kevinburke** [comment]
Ah thanks for the feedback, should be updated now. There's one set that's 0x2C which seems odd, no other code references the flag at 1 << 6 or sets three flags

**Comment by IntelOrca** [comment]
```
0x2C = 0x20 | 0x8 | 0x4
     = RIDE_INVALIDATE_RIDE_MAIN | RIDE_INVALIDATE_RIDE_LIST | RIDE_INVALIDATE_RIDE_MAINTENANCE
```

**Comment by kevinburke** [comment]
Ahh right... okay I updated it.

**Pull request merged**